src/header/styles.scss

```diff
@@ -45,7 +46,7 @@
 }
 
 .main {
-  @include styles.text-flex-wrapping;
```

Comment:
changed to general `text-wrapping`, as `flex-wrapping` has `overflow: hidden`, which hides focus outline

Reply:
I would imagine the style was there for a reason. These styles also have different min/max widths. You can run the screenshot tests to see if there are some regressions. Ideally is to test the before and after with some large contents and see if all good.